### DISCUSSION TITLE
vscode公式预览有奇怪的字符

### DISCUSSION THREAD

**Discussion by lcq-molybdenum42** [sjtug/SJTUThesis#1071] [提问] [answered]
![image](https://github.com/user-attachments/assets/4224bcbb-9af8-4a29-b563-9b0ff3d79fea)
如图，鼠标悬停在公式代码上时弹出的预览中出现了“\DeclareRobustCommand\cs[1]\char'\ #1”的前缀，不知道怎么去除，求大佬路过解答！！

**Comment by NemoYuan2008** [comment, marked as answer]
这似乎是 VScode 插件的 bug，不过，你可以将 `setup.tex` 中 
```
\DeclareRobustCommand\cs[1]{\texttt{\char`\\#1}}
```
这一行删掉，这样就不会有问题了。

这一行定义的命令在编写模板中的示例内容时用到了，但如果你已经删掉了原有的示例内容，即可安全去掉这一行。

**Reply by lcq-molybdenum42** [reply]
感谢解答！
注释掉这一句之后示例内容确实报错了，但是求助gpt把新定义的命令改回原本的样子，就可以保留示例内容了。
